src/hooks/useJurorSubscriptionFees.js

```diff
@@ -0,0 +1,44 @@
+import { useEffect, useState } from 'react'
+import { useWallet } from 'use-wallet'
+import { useCourtSubscriptionActions } from './useCourtContracts'
+
+export default function useJurorSubscriptionFees() {
+  const [subscriptionFees, setSubscriptionFees] = useState([])
+  const wallet = useWallet()
+  const { getters } = useCourtSubscriptionActions()
+
+  useEffect(() => {
+    const fetchSubscriptionFees = async () => {
+      if (!getters) {
+        return
+      }
+
+      try {
+        const currentPeriodId = await getters.getCurrentPeriodId()
+
+        const jurorSubscriptionsFees = []
+        for (let periodId = 0; periodId < currentPeriodId; periodId++) {
+          const jurorShare = await getters.getJurorShare(
+            wallet.account,
+            periodId
+          )
+
+          if (
+            jurorShare[1].gt(0) &&
```

Comment:
maybe add some comments here explaining why we are getting the jurorsShare at `[1]`?

Reply:
Yes good idea